examples/go-client/configuration/EnvConfig.go

```diff
@@ -0,0 +1,61 @@
+// +build sample_client
+
+
+package configuration
+
+import "fmt"
+import "github.com/kelseyhightower/envconfig"
+
+type AppEnvConfigPoc struct {
+	EngineEnvConfig
+	GenericEnvConfig
+}
+
+type EngineEnvConfig struct {
```

Comment:
Seems like we shouldn't repeat this kind of `struct` here when we have (for example) `sys. SystemConfig`. Can you refactor to use existing structs?